Content.Server/Headset/HeadsetComponent.cs

```diff
@@ -63,6 +63,18 @@ public void Receive(string message, RadioChannelPrototype channel, EntityUid sou
                 MessageWrap = Loc.GetString("chat-radio-message-wrap", ("color", channel.Color), ("channel", $"\\[{channel.LocalizedName}\\]"), ("name", _entMan.GetComponent<MetaDataComponent>(source).EntityName))
             };
 
+            // If the source isn't a player, we set the chat message accordingly
+            if (announcement)
+            {
+                msg = new MsgChatMessage
+                {
+                    Channel = ChatChannel.Radio,
+                    Message = message,
+                    //Square brackets are added here to avoid issues with escaping
+                    MessageWrap = Loc.GetString("announcement-radio-message-wrap", ("color", channel.Color), ("channel", $"\\[{channel.LocalizedName}\\]"))
+                };
+            }
```

Comment:
Should be if else as it creates an unnecessary message.

Reply:
Could you explain further? Right now, a `MsgChatMessage` is created outside of the if statement and is simply overridden, do you want me to have it create a blank `MsgChatMessage` and have it define the value with if/else?

Reply:
The initial message may be redundant and I'm not sure if the compiler would optimise it out so better to just make it an if-else to make the intend clearer.